data/remnant missions.txt

```diff
@@ -493,5 +597,772 @@ mission "Remnant: Technology Available"
+			choice
+				"It isn't that important."
+					decline
+				"I should find a prefect."
+			`	For lack of a better place to go, you stop by the information desk where you picked up the sign language course. You tell the attendant that you have something potentially important - but non-urgent - to discuss in regards to Remnant security. They tap a few buttons, then give you directions down a hallway. You quickly find yourself in an armory where several remnant guards are in the process of cleaning their weapons. One stands up and comes over to you.`
+			`	"Greetings Captain, I am prefect Reegar. I was told you had some information that could be important?"`
+			`	He takes a seat on a bench a little ways away from the other Remnant, and gestures for you to take a seat with him.`
+			`	You give him some background about the current state of Human and Hai understanding about the wormholes - or rather the lack thereof, then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system. He quickly picks up on the potential threat if someone figures out the key to using them, especially since you have demonstrated that it only takes one curious human to figure it out.`
+			`	Thinking carefully, he asks "It is clear that we need to keep an eye on this situation. You know more about the rest of the galaxy than we do, could you find them and see how much progress they have made?"`
+			choice
+				"I'd prefer not to."
+					decline
+				"I'd be happy to."
+			`	"Excellent. See what you can find out about their research and results, but it would probably be wise to avoid making direct contact with them."`
+				accept
+	on complete
+		conversation
+			` Arriving on Bounty, you take a discrete look around the spaceport, but you don't find the researchers or any sign of them. It appears that they packed up and left since the last time you saw them. Time to look around for some info.`
+
+mission "Remnant: Follow-up on Terminus Researchers 2"
+	name "Check up on Terminus Researchers"
+	description "Tracking down the wormhole researchers."
+	source Bounty
+	destination
+		government Deep
+	to offer
+		has "Remnant: Follow-up on Terminus Researchers: done"
+	on offer
+		conversation
+			`A few hours of searching later, you find a lead: It seems that launching probes into the Terminus system isn't exactly a common activity, so their permits stand out from the rest. A helpful clerk in the spaceport office pulled up their application and found that they are listed as part of a university in the Deep, although it doesn't mention which planet their specific campus is on. You pull up a list of Deep worlds and sigh. Looks like you'll have to play tourist for a few weeks.`
```

Comment:
Preference, but I wouldn't use a period instead of the colon here.

The last two sentences don't fit Endless Sky's tone, which should not assume the player's thoughts in such a non-narrative manner.